package.json

```diff
@@ -146,32 +146,31 @@
   "devDependencies": {
     "@types/adm-zip": "^0.4.32",
     "@types/dotenv": "6.1.1",
-    "@typescript-eslint/eslint-plugin": "1.10.2",
-    "@typescript-eslint/parser": "1.10.2",
-    "@vitalets/google-translate-api": "2.8.0",
-    "assert": "1.4.1",
-    "cp-file": "5.0.0",
+    "@typescript-eslint/eslint-plugin": "2.3.2",
+    "@typescript-eslint/parser": "2.3.2",
+    "@vitalets/google-translate-api": "3.0.0",
+    "assert": "2.0.0",
+    "cp-file": "7.0.0",
     "devtron": "1.4.0",
     "electron": "3.1.10",
-    "electron-builder": "20.43.0",
-    "electron-connect": "0.6.2",
-    "electron-debug": "1.4.0",
-    "electron-notarize": "0.2.0",
-    "eslint-config-xo-typescript": "0.14.0",
+    "electron-builder": "21.2.0",
```

Comment:
@SamuelMarks please revert this. We don't upgrade electron-builder like this since there is heavy testing involve in this.